src/Log/ValueObjects/LogType.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Give\Log\ValueObjects;
+
+/**
+ * Class LogType
+ * @package Give\Log\ValueObjects
+ *
+ * @since 2.9.7
+ */
+class LogType {
```

Comment:
Same thing here. I am a fan of Value Objects, but make sure it's actually an instantiable object that contains and validates the value.

Reply:
Resolved a89ff89a